src/generator/models.ts

```diff
@@ -13,6 +13,7 @@ import { ImportManager } from './imports';
 
 // Creates the content in models.go
 export async function generateModels(session: Session<CodeModel>): Promise<string> {
+  imports.clear();
```

Comment:
It would be cool if we could consolidate this in one place. Not saying it's possible with the code as it is at the moment, but maybe later on. It might be easy to forget to clear it in two different places.

Reply:
It's only this way because we use a package-local variable.  If we refactored to remove that, the need for `clear()` would go away.  However, it does mean we have to pass around the instance which is cumbersome.

Reply:
I refactored the code to remove the need for `clear()`